src/link/dto/link.dto.ts

```diff
@@ -0,0 +1,61 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsNotEmpty } from "class-validator";
+
+
+export class ElderlyCodeDto {
+    @ApiProperty()
+    @IsNotEmpty() 
+    code: string;
```

Comment:
Adding class validator type here can make it better.

Ex; @IsString()
@IsNumber()